### ISSUE TITLE
Server pagination: currentPage can't be set as starting page

### ISSUE THREAD

**Issue by lochstar** [njleonzhang/vue-data-tables#172]
When setting `paginationDef.currentPage` on `data-tables-server`, if I've selected a value greater than 1, it is not honoured in the pagination. It appears to work with normal `data-tables`.

See this fiddle: https://jsfiddle.net/v0youfw5/

Thanks

**Comment by njleonzhang** [comment]
Tnx, will fix in next version.

**Comment by njleonzhang** [comment]
You can preview next version, refer to https://github.com/njleonzhang/vue-data-tables/issues/170

**Comment by njleonzhang** [comment]
You can try the beta version. 
```
 npm install vue-data-tables@beta
```

main changes:

* remove the default action bar and encourage use customize action bar.
* improve render performance with filter function cache.
* sync modifier for currentPage and pageSize

**Comment by njleonzhang** [comment]
things work in my dev environment, but fails after release. I will check if there is something wrong with my webpack bundle config, will keep you update.

**Comment by njleonzhang** [comment]
version 3.4 is released.

**Comment by lochstar** [comment]
I've noticed a bug when navigating back to a page with data-tables-server. I am ignoring the "init" event that is fired, as the data has already been fetched. In this case, the pagination total is not set correctly.

I've identified the bug here: https://github.com/njleonzhang/vue-data-tables/blob/97df80e64860c3cf14d0badf5abc6999bb458fb7/src/components/DataTablesServer.vue#L35

I believe it should be `let ceilTotalPage = Math.ceil(totalPage)`. I as going to submit a PR but run in to problems running the development server.
